src/test/a11y/a11y.ts

```diff
@@ -21,13 +21,26 @@ class PallyIssue {
   typeCode: number
 }
 
-describe('Accessibility', () => {
+function ensurePageCallWillSucceed (url: string): Promise<void> {
+  return agent.get(url)
```

Comment:
Do we need to handle post calls as well? Not sure if we use them in our application. 
pa11y('http://example.com/', {
    headers: {
        'Content-Type': 'application/json'
    },
    method: 'POST',
    postData: '{"foo": "bar", "bar": "baz"}'
});

Reply:
@mounikahmcts Interesting. Correct me if i'm wrong but a11y is just to test the accessibility of a view no? Should or does a11y care about data in a post?

Reply:
I think a11y is just accessibility but we will test post calls in our route tests.

Reply:
just to add, i just fixed lint issues in this file as part of this PR. I didn't do anything else :).

Reply:
@Jamesnelson117  @KiranVarma02  I meant testing a page which can be rendered only via post call. But yeah, we can cover them in route tests if the scenario comes